cmd/notary/prettyprint.go

```diff
@@ -198,6 +198,19 @@ func prettyPrintRoles(rs []*data.Role, writer io.Writer, roleType string) {
 	table.Render()
 }
 
+// Pretty-prints a list of delegation paths, and ensures the empty string is printed as "" in the console
+func prettyPrintPaths(paths []string) string {
+	prettyPaths := []string{}
+	for _, path := range paths {
+		// manually escape "" and designate that it is all paths with an extra print <all paths>
+		if path == "" {
+			path = "\"\" <all paths>"
```

Comment:
Totally non-blocking, and this may already be done elsewhere, but I probably missed it - would sorting the paths make it easier for a user to see if they have overlapping path permissions?  So the all paths would always be in front, and then paths that overlap each other would be right next to each other?